core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -1011,10 +996,25 @@ private Builder(TableMetadata base) {
       this.specsById = Maps.newHashMap(base.specsById);
       this.sortOrdersById = Maps.newHashMap(base.sortOrdersById);
 
-      this.rowLineage = base.rowLineageEnabled;
       this.nextRowId = base.nextRowId;
     }
 
+    /**
```

Comment:
nit: Not sure why this is registered as net new, did it get moved? Doesn't really matter since we are deprecating